roles/swift-object/defaults/main.yml

```diff
@@ -2,4 +2,7 @@
 swift_object:
   ip: "{{ primary_ip }}"
   port: 6000
-  workers: 10
+  workers: "{{ swift.cluster.no_workers }}"
+  replicators: "{{ swift.cluster.no_replicators }}"
+  auditors: "{{ swift.cluster.no_auditors }}"
+  updaters: 2
```

Comment:
Lets just pin it to the other updaters for consistency sake, and see if it still meets our performance needs.